src/plugins/utils/utils-object.ts

```diff
@@ -31,7 +31,7 @@ export function deepFreeze<T>(o: T): T {
  * RxDB normally uses the 'dot-prop' npm module.
  * But when performance is really relevant, this is not fast enough.
  * Instead we use a monad that can prepare some stuff up front
- * and we can re-use the generated function.
+ * and we can reuse the generated function.
```

Comment:
lol, those damn linters eh

Reply:
new version of Jetbrains forces a plugin on you that 'corrects' spelling. i dislike it, it's often wrong or pedantic. maybe now i know how it feels 😂